fhir-parent/pom.xml

```diff
@@ -616,4 +622,12 @@
             </properties>
         </profile>
     </profiles>
+
+    <repositories>
+        <repository>
+            <id>github</id>
+            <name>GitHub lmsurpre Apache Maven Packages</name>
+            <url>https://maven.pkg.github.com/lmsurpre/fhir</url>
```

Comment:
should we put to ibm/fhir?

Reply:
good catch.  this pipeline doesn't even put to the package registry yet and so this PR shouldn't have included this.  will clean it out now.